index.js

```diff
@@ -139,10 +140,7 @@ export const startEvaluate = async ({
   })
 
   // Update top measurement stations every 12 hours
-  const pgClient = await createPgClient()
-  setInterval(async () => {
-    await pgClient.query('REFRESH MATERIALIZED VIEW top_measurement_participants_yesterday_mv')
-  }, 1000 * 60 * 60 * 12)
+  setInterval(async () => { await updateTopMeasurementStations(createPgClient) }, 1000 * 60 * 60 * 12)
```

Comment:
Nitpick: there is no need to wrap the function call in another async function and `await`.

```suggestion
  setInterval(() => updateTopMeasurementStations(createPgClient), 1000 * 60 * 60 * 12)
```

Reply:
Agreed

Reply:
Addressed in https://github.com/filecoin-station/spark-evaluate/pull/278/commits/3b8b7416a86cb7cfe21fd52ca6f45c8c8b8c6f08